docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -226,6 +226,40 @@ defs = Definitions(
 )
 ```
 
+### Checks on partitioned assets
+
+Checks are supported per-asset, not per-partition. To write a check on a partitioned asset, you'll want to summarize the status of all the partitions in to one pass/fail for the asset.
```

Comment:
> To write a check on a partitioned asset, you'll want to summarize the status of all the partitions **in to one pass/fail for the asset.**

I'm not sure why, but the last phrase here kept throwing me off - I think it's the `in to one` that's doing it. What would you think about something like:

> To write a check on a partitioned asset, summarize the status of all the partitions as a single pass/fail for the asset.